### PR TITLE
[COM-1447] Setup nx

### PR DESCRIPTION
**Pull request by guilean**
[Ticket](https://impulsumvc.atlassian.net/browse/COM-1447)

### CODE REVIEW

File: LICENSE
Comment (tkanzakic) [inline LICENSE:1]: should we restore this?

File: apps/nextjs-contentful/src/pages/[slug].tsx [hunk 1]
```diff
@@ -4,11 +4,13 @@ import { pageRepository } from '../repositories/page';
 import { pathsRepository } from '../repositories/paths';
 
 export const getStaticPaths: GetStaticPaths = async () => ({
-  paths: (await pathsRepository.getStaticSlugs()).map((slug: string) => ({ params: { slug } })),
+  paths: (await pathsRepository.getStaticSlugs()).map((slug: string) => ({
+    params: { slug },
+  })),
```
Comment (tkanzakic) [inline apps/nextjs-contentful/src/pages/[slug].tsx:9]: why has this changed, is it because of the changes in the `.editorconfig`?

File: .editorconfig
Comment (tkanzakic) [inline .editorconfig:1]: why this changes?

File: apps/nextjs-contentful/src/repositories/paths.test.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import { pathsRepository } from './paths';
 
-const mockGetStaticRoutes = jest.fn().mockResolvedValue([{ slug: '/slug1' }, { slug: '/slug2' }]);
+const mockGetStaticRoutes = jest
+  .fn()
+  .mockResolvedValue([{ slug: '/slug1' }, { slug: '/slug2' }]);
```
Comment (tkanzakic) [inline apps/nextjs-contentful/src/repositories/paths.test.ts]: ditto

File: package-lock.json
Comment (tkanzakic) [inline package-lock.json:1]: why do we have a package-lock and a yarn.lock?

File: apps/nextjs-contentful/src/pages/[slug].tsx (hunk 1 above)
Comment (guilean) [reply apps/nextjs-contentful/src/pages/[slug].tsx:9]: Most changes are due to the config of the editorconfig with the exception of the parameters passed to the `getStaticPageContent` method

File: package-lock.json
Comment (guilean) [reply package-lock.json:1]: I've removed the `yarn.lock` in favor of the `package-lock` so we can transition from `yarn` to use `npm` as the package manager

File: .editorconfig
Comment (guilean) [reply .editorconfig:1]: My bad, restored original configuration

File: package-lock.json
Comment (tkanzakic) [reply package-lock.json:1]: sounds good

**Review by tkanzakic — APPROVED** [review]
